policy/warn-k8s-namespace-conftestcombine-bestpractices.rego

```diff
@@ -0,0 +1,21 @@
+package main
+
+warn[msg] {
+  manifests := input[_]
+  some i
+
+  manifests[i].apiVersion == "v1"
+  manifests[i].kind == "Namespace"
+  namespace := manifests[i]
+
+  not namespaceHasNetworkPolicy(namespace, manifests)
```

Comment:
@garethahealy - does `namespace := manifests[i]` pop the item out of the array so it's no longer in `manifests `?

Reply:
no, just sets the var

Reply:
oh ok - so namespace is not needed in your function below

Reply:
not currently, but if the policy was to be extended to match selectors, then yep it would

Reply:
That's grand, i'll merge this one in now so.